src/coreneuron/permute/node_permute.cpp

```diff
@@ -107,20 +107,17 @@ namespace neuron {
 #endif
 
 #if !CORENRN_BUILD
-static int nrn_soa_padded_size(int cnt, int layout) {
-    assert(layout == 1);
+static int nrn_soa_padded_size(int cnt) {
     return cnt;
 }
-static int nrn_i_layout(int icnt, int cnt, int isz, int sz, int layout) {
+static int nrn_i_layout(int icnt, int cnt, int isz) {
```

Comment:
I understand that `sz` isn't needed, but it's an unrelated change. Not sure it's worth splitting.